data/json/body_parts.json

```diff
@@ -2,9 +2,9 @@
   {
     "id": "torso",
     "type": "body_part",
-    "name": "torso",
-    "heading_singular": "Torso",
-    "heading_plural": "Torso",
+    "name": { "str": "torso", "str_pl": "torsos" },
+    "name_accusative": { "str": "torso", "str_pl": "torsos" },
```

Comment:
@Qrox: Can you please advice if I will need to use a `ctxt` here? It was used as `return pgettext( "bodypart_accusative", bdy.name_multiple.c_str() );` previously.